review/get_review_ids.py

```diff
@@ -49,16 +49,20 @@ def get_problems(num_desired, current_course):
     store = modulestore()
     course_usage_key = store.make_course_usage_key(current_course)
     course_blocks = get_course_blocks(user, course_usage_key)
+    review_course = current_course.replace(course=current_course.course+'_review')
+    review_course_usage_key = store.make_course_usage_key(review_course)
+    review_course_blocks = get_course_blocks(user, review_course_usage_key)
```

Comment:
When does this issue occur?  If problems are added to the course after the review course is created?  How likely is that scenario?
You are trading off performance versus correctness here.  This extra call to `get_course_blocks` adds an extra cost.  For large courses, it may take 500ms.

Reply:
The scenario you described is fairly likely since course teams often are changing their course content. Since they would now need to update the course in two areas (actual and review course), there is going to be a lag during which the XBlock could try and load a problem that only exists in one of the two courses. It ends up loading a 404 inside of the XBlock leading to a bad UX.